pkg/infrastructure/clusterapi/clusterapi.go

```diff
@@ -173,7 +190,7 @@ func (i *InfraProvider) Provision(ctx context.Context, dir string, parents asset
 			cluster = c
 			return cluster.Status.InfrastructureReady, nil
 		}); err != nil {
-			return fileList, err
+			return fileList, fmt.Errorf("infrastructure was not ready within %v: %w", timeout, err)
```

Comment:
What about:
```suggestion
            if wait.Interrupted(err) {
			    return fileList, fmt.Errorf("infrastructure was not ready within %v", timeout)
			}
			return fileList, err
```

Reply:
Otherwise it reads like we're always timing out even when we fail immediately?

Reply:
perfect, thanks!